### DISCUSSION TITLE
Trigger action on value-change of specific fields

### DISCUSSION THREAD

**Discussion by webwurm** [AntlerVC/firetable#937] [Support / Q&A] [answered]
I have a basic question for which I can't find the answer:

With a derivative field I can easily say it is triggered when certain fields change:
<img width="350" alt="image" src="https://user-images.githubusercontent.com/1053176/201105492-d3d78e17-ff3b-4590-b313-3947ab3fe849.png">

How do I do that with an action? I don't want to manually trigger it, but also, when a specific field(s) in the same row changes.

What is the best/inteded way to do this?

Thanks!

**Comment by shamsmosowi** [comment, marked as answer]
potentially using a `task` extension could be what you're after 

<img width="795" alt="Screen Shot 2022-11-10 at 3 25 38 pm" src="https://user-images.githubusercontent.com/13803906/201117309-39d438ea-69a9-4c4c-8677-3628f2150e1a.png">

**Comment by webwurm** [comment]
Thanks, that was exactly what I was looking for!
